spec/lib/krikri/registry_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+
+describe Krikri::Registry do
+
+  before(:all) do
+    described_class.register('mock', Class.new)
```

Comment:
See [here](https://github.com/dpla/KriKri/pull/46/files#r22553261) and [here](https://github.com/dpla/KriKri/pull/46/files#r22553388).